installer.sh.in

```diff
@@ -1187,7 +1238,9 @@ install_packages() {
 }
 
 enable_dhcpd() {
-    ln -sf /etc/sv/dhcpcd $TARGETDIR/etc/runit/runsvdir/default/dhcpcd
+    ln -sf /etc/sv/dhcpcd $TARGETDIR/etc/runit/runsvdir/default
+    ln -sf /etc/sv/wpa_supplicant $TARGETDIR/etc/runit/runsvdir/default
+    cp -r /etc/wpa_supplicant/* $TARGETDIR/etc/wpa_supplicant
 }
```

Comment:
And if they don't want wifi?

Reply:
True. wpa_supplicant should be enabled somewhere else in the installer.